src/lib/components/Feedback/index.js

```diff
@@ -0,0 +1,202 @@
+                  value="0"
+                />
+              </label>
+            </div>
+
+            <div class="web-feedback__row">
+              <div id="effective-label">
+                <label>Did this page help you complete your goal(s)?</label>
+              </div>
+              <label>
+                <input
+                  type="radio"
+                  aria-labelledby="effective-label yes-label"
+                  name="Effectiveness"
+                  value="1"
+                />
+              </label>
+              <label>
+                <input
+                  type="radio"
+                  aria-labelledby="effective-label no-label"
+                  name="Effectiveness"
+                  value="0"
+                />
+              </label>
+            </div>
+
+            <div class="web-feedback__row">
+              <div id="complete-label">
+                <label>Did this page have the information you needed?</label>
+              </div>
+              <label>
+                <input
+                  type="radio"
+                  aria-labelledby="complete-label yes-label"
+                  name="Completeness"
+                  value="1"
+                />
+              </label>
+              <label>
+                <input
+                  type="radio"
+                  aria-labelledby="complete-label no-label"
+                  name="Completeness"
+                  value="0"
+                />
+              </label>
+            </div>
+
+            <div class="web-feedback__row">
+              <div id="accuracy-label">
+                <label>Was this page's information accurate?</label>
+              </div>
+              <label>
+                <input
+                  type="radio"
+                  aria-labelledby="accuracy-label yes-label"
+                  name="Accuracy"
+                  value="1"
+                />
+              </label>
+              <label>
+                <input
+                  type="radio"
+                  aria-labelledby="accuracy-label no-label"
+                  name="Accuracy"
+                  value="0"
+                />
+              </label>
+            </div>
+
+            <div class="web-feedback__row">
+              <div id="readable-label">
+                <label>Was this page easy to read?</label>
+              </div>
+              <label>
+                <input
+                  type="radio"
+                  aria-labelledby="readable-label yes-label"
+                  name="Readability"
+                  value="1"
+                />
+              </label>
+              <label>
+                <input
+                  type="radio"
+                  aria-labelledby="readable-label no-label"
+                  name="Readability"
+                  value="0"
+                />
+              </label>
+            </div>
+          </div>
+
+          <div class="w-text--center">
```

Comment:
Maybe give this element `.5em` of `padding-top` so it has a bit of breathing room on mobile.